README.md

```diff
@@ -37,7 +37,7 @@ The developer [mailing list](https://lists.linuxfoundation.org/mailman/listinfo/
 should be used to discuss complicated or controversial changes before working
 on a patch set.
 
-Developer IRC can be found on Freenode at #bitcoin-core-dev.
+Developer IRC can be found on Freenode at [#bitcoin-core-dev](https://webchat.freenode.net/?channels=#bitcoin-core-dev).
```

Comment:
I am not convinced we should mention the dev chat in the readme...

We already link to CONTRIBUTING.md just three lines above. Do you mind to move the `Developer IRC can be found on Freenode at #bitcoin-core-dev. ` to CONTRIBUTING.md ?